otx/mpa/deploy/utils/utils.py

```diff
@@ -26,8 +38,27 @@ def convert_batchnorm(module):
         module_output.num_batches_tracked = module.num_batches_tracked
 
```

Comment:
```
if hasattr(module, "qconfig"):
    module_output.qconfig = module.qconfig
```
How do you think about adding these lines at this place? I found that this part exists in `convert_sync_batchnorm()` in torch.

Reply:
Thanks for the suggestion. Added in https://github.com/openvinotoolkit/training_extensions/pull/1659/commits/1753494e46ff2b2936d9582d34670b06e72c5a92.